CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## New features and fixes
 - Provide a generator function to collect `mdata` in `run!` and `ensemblerun!`.
 - Save/load entire models using `save_checkpoint` and `load_checkpoint`
+- New functions `get_spatial_property` and `get_spatial_index` that allows better handling of spatial fields present in `ContinuousSpace` that are represented via the forms of discretization over the space.
```

Comment:
New functions `get_spatial_property` and `get_spatial_index`, which provide better handling of discrete representations of spatial fields present in `ContinuousSpace`.

Reply:
that's already from my PR